src/animation/AnimatedScaleComponent.js

```diff
@@ -2,9 +2,12 @@
 import React, { PureComponent } from 'react';
 import { Animated, Easing } from 'react-native';
 
+import type { StyleObj } from '../types';
+
 type Props = {
   children: [],
   visible: boolean,
+  wrapperStyle?: StyleObj
```

Comment:
Just call it `style`. We already do that for other components.

Reply:
Done 👍